### ISSUE TITLE
no mongo/mongoose?

### ISSUE THREAD

**Issue by sw-yx** [cllgeek/mern-boilerplate#1]
i typically take MERN to mean Mongo + Express + React + Node. This boilerplate doesnt use mongo. what does your M stand for?

**Comment by cllgeek** [comment]
@sw-yx yes,you are right. there are M stand for mongod,but this boilerplate doesnt hava it. I forget to adding it,sorry,I'm going to add it for a while,thank you.

**Comment by sw-yx** [comment]
ok cool. please let me know when you have it!
